cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -213,6 +213,9 @@
     umount /var/lib/snapd/hostfs/proc/,
     mount options=(rw rslave) -> /var/lib/snapd/hostfs/,
 
+    # Allow reading the os-release file (possibly a symlink to /usr/lib).
+    /{etc/,usr/lib/}os-release r,
```

Comment:
What puzzles me is why we did not notice this earlier, I think we have a missing (spread) test here :/